src/transformations/correction/valid_point.jl

```diff
@@ -0,0 +1,3 @@
+# # Valid Point
+
+# A point is valid if it has no `NaN` or `Inf` values.
```

Comment:
Did we want code here? Or is this just setup for the next PR?

Reply:
Ah this was just setup, but I can remove it for a cleaner PR.  This one was mainly for the API and structure - adding more should be simpler afterwards.